etcdutl/snapshot/v3_snapshot.go

```diff
@@ -311,6 +333,79 @@ func (s *v3Manager) saveDB() error {
 	return nil
 }
 
+// revBump increases the revision, modrevision, and version of every key to force watchers to resync.
+//
+// In order to ensure that watchers don't have a revision that's higher than the restored revision,
+// 'amount' should be a very large number, > 1_000_000.
```

Comment:
1`000`000 is not a large number for revision. Restored backup could have couple of days. 1 QPS for 11 days is enough to hit 1000000. Please suggest at least 1`000`000`000, even with such large bump it's enough to make 9`223`372`036 backups.